Card/build.gradle

```diff
@@ -34,7 +34,6 @@ android {
 
 dependencies {
     api project(':BraintreeCore')
-    implementation project(':BraintreeDataCollector')
```

Comment:
How is it that we can safely remove this data collector without needing a replacement? If this is something I'm overlooking I blame that I don't have enough caffeine this morning 🤣

Reply:
I really appreciate you bringing this up actually. In a strict sense for a merchant that has Kount enabled I guess it actually would result in a behavioral breaking change – only the API is unchanged. I wonder if maybe we should defer the change in that case for `Card`, `LocalPayment` and `PayPal` modules?

Reply:
Actually i'm mistaken in this case. The `:Card` module wasn't actually performing data collection so it made sense to remove it entirely.

Since this dependency is marked as `implementation` it was never added to the merchant app's Java class path, and if a merchant is minifying their release build it should be optimized out since we aren't using it within the SDK.